react-native-purchases-ui/src/index.tsx

```diff
@@ -308,15 +361,96 @@ export default class RevenueCatUI {
     );
   };
 
-    /**
+  /**
    * Presents the customer center to the user.
    * 
+   * @param {CustomerCenterCallbacks} callbacks - Optional callbacks for customer center events.
    * @returns {Promise<void>} A promise that resolves when the customer center is presented.
    */
-    public static presentCustomerCenter(): Promise<void> {
-      return RNCustomerCenter.presentCustomerCenter();
+  public static presentCustomerCenter(callbacks?: CustomerCenterCallbacks): Promise<void> {
```

Comment:
What do you think about making the parameters an object? Like
```suggestion
  public static presentCustomerCenter(params?: { callbacks?: CustomerCenterCallbacks }): Promise<void> {
```

Or maybe even extract the type of the params out... Seems like there might be more parameters we would like to add in the future, and this way it would be much more extensible IMO.